src/types/redis_timeseries.h

```diff
@@ -173,6 +174,25 @@ struct TSRangeOption {
   BucketTimestampType bucket_timestamp_type = BucketTimestampType::Start;
 };
 
+struct TSMGetOption {
+  struct FilterOption {
+    std::unordered_map<std::string, std::set<std::string>> labels_equals;
+    std::unordered_map<std::string, std::set<std::string>> labels_not_equals;
+    std::set<std::string> labels_exists;
+    std::set<std::string> labels_not_exists;
```

Comment:
Could labels_equals {"label", empty set} represent labels_not_exists "label"? So that we can use just two fields for the filter.